Lombiq.Tests.UI.Samples/Tests/DataBaseSnapshotTests.cs

```diff
@@ -0,0 +1,43 @@
+using Lombiq.Tests.UI.Attributes;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Samples.Constants;
+using Lombiq.Tests.UI.Services;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Lombiq.Tests.UI.Samples.Tests;
+
+// We can execute tests on already existing databases. To demo this, we will take a snapshot of the running application,
+// and use that, because we don't have a pre-beaked database. Normally this feature is used to run tests when there is
+// an already existing database, so you don't need to take a snapshot before using the
+// "ExecuteTestFromExistingDBAsync()" method.
+public class DatabaseSnapshotTests : UITestBase
```

Comment:
The file name remained wrong.

Reply:
Well, that is strange: 
![image](https://user-images.githubusercontent.com/80963259/167025445-13670a64-bd6b-4119-8e90-82a208eb1795.png)

For some reason GitExtensions didn't detected the file name change (I guess because it was just casing). Fixed now.